app/models/legal_aid_application.rb

```diff
@@ -105,6 +105,10 @@ def other_assets?
     other_assets_declaration.present? && other_assets_declaration.positive?
   end
 
+  def proceeding_type?(type)
+    proceeding_types.any? { |proceeding_type| proceeding_type[:ccms_matter] == type }
```

Comment:
proceeding_type is an object, not a hash, so you can use `proceeding_type.ccms_matter` instead of `proceeding_type[:ccms_matter]`